resources/assets/js/models/jobUtil.ts

```diff
@@ -99,3 +102,22 @@ export const getSkillLevelName = (
   }
   return skillLevelName(skill_level_id, skill_type_id);
 };
+
+// TODO: allow for Complete status.
+export const jobStatus = (job: Job): JobStatus => {
```

Comment:
Shouldn't the server be deciding this?

Reply:
Yeah, you're right. We'll be changing the status system more in a future sprint, but for now I could just add an accessor to the JobPoster model that returns this status id using the same logic. Do you think that's okay for now?

Reply:
If we aren't yet keeping the `job_poster_status` table in sync with the `JobPoster` model's `status()` method, then yes, we should add something with this logic on the PHP side to be consumed by Javascript.